languagetools/build.gradle

```diff
@@ -0,0 +1,86 @@
+plugins {
+    id 'java'
+}
+repositories {
+    mavenCentral()
+}
+
+def  languageToolVersion = '6.1'
+def plugins = [
+        [ lang: 'Arabic', code: 'ar', className: 'ArabicPlugin'],
+        [ lang: 'Asturian', code: 'ast', className: 'AsturianPlugin'],
+        [ lang: 'Belarusian', code: 'be', className: 'BelarusianPlugin'],
+        [ lang: 'Breton', code: 'br', className: 'BretonPlugin'],
+        [ lang: 'Catalan', code: 'ca', className: 'CatalanPlugin'],
+        [ lang: 'Danish', code: 'da', className: 'DanishPlugin'],
+        [ lang: 'German', code: 'de', className: 'GermanPlugin'],
+        [ lang: 'Greek', code: 'el', className: 'GreekPlugin'],
+        [ lang: 'English', code: 'en', className: 'EnglishPlugin'],
+        [ lang: 'Esperanto', code: 'eo', className: 'EsperantoPlugin'],
+        [ lang: 'Spanish', code: 'es', className: 'SpanishPlugin'],
+        [ lang: 'Percian', code: 'fa', className: 'PercianPlugin'],
```

Comment:
Typos:
- Percian --> Persian
- PercianPlugin --> PersianPlugin

This also affects other files.